apps/yarn.lock

```diff
@@ -2897,10 +2897,6 @@ asn1@~0.2.3:
   version "0.2.3"
```

Comment:
We drop 32 dependencies as a result of this change, shrinking `node_modules` by... about 8k. :joy: